openedx/adg/lms/applications/forms.py

```diff
@@ -30,6 +30,7 @@ class ExtendedUserProfileForm(forms.Form):
     birth_month = forms.IntegerField()
     birth_year = forms.IntegerField()
     saudi_national = forms.BooleanField(required=False)
+    how_did_you_hear_about_this_program = forms.CharField(required=False)
```

Comment:
We can add the max length here and then validation would be taken care automatically.